app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class TweetsController < ApplicationController
+  enable :sessions
+  set :session_secret, 'fwitter'
+  set :public_folder, 'public'
+  set :views, 'app/views'
+
+# Shows Tweets index page
+  get '/tweets' do
+    redirect '/login' if !logged_in?
+    @tweets = Tweet.all
+    erb :'/tweets/tweets'
+  end
+
+# Shows the "New Tweet" form and makes a new tweet
+  get '/tweets/new' do
+    redirect '/login' if !logged_in?
+    erb :'/tweets/new'
+  end
+
+  post '/tweets' do
+    redirect '/login' if !logged_in?
+    redirect '/tweets/new' if params[:tweet][:content].empty?
+    @tweet = Tweet.create(params[:tweet])
+    @tweet.user_id = User.find_by(username: current_user).id
+    @tweet.save
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```